guides/common/modules/con_upgrading-project.adoc

```diff
@@ -1,4 +1,4 @@
 [id="Upgrading_{project-context}_{context}"]
 = Upgrading {ProjectName}
 
-Use the following procedures to upgrade your existing {ProjectName} to {ProjectName} {ProjectVersion}.
+You can upgrade from any patch version of {ProjectServer} {ProjectVersionPrevious} to {ProjectServer} {ProjectVersion}.
```

Comment:
@evgeni Maximilian pointed out this line in his review (and also line 7 in `guides/common/modules/proc_upgrading-smartproxy-server.adoc`).

For context, this is based on the current version of the upgrade guide where in another (now deleted) section, we show this: https://github.com/theforeman/foreman-documentation/blob/2b04b6d2bae80df1d85a38ce9b18b7f2db514039/guides/common/modules/con_project-server-upgrade-considerations.adoc?plain=1#L5

Can you confirm that the current statement about upgrading "from any patch version" being possible is okay?

Reply:
Not @evgeni but it really is `patch`, not `minor`.

Reply:
If the question is "are all of the following upgrades allowed?", then the answer is yes:
- Foreman 3.11.0 to 3.12(.latest)
- Foreman 3.11.4 to 3.12(.latest)
- Katello 4.14.0 to 4.15(.latest)
- Katello 4.14.3 to 4.15(.latest)
- Satellite 6.16.0 to 6.17(.latest)
- Satellite 6.16.5.1 to 6.16(.latest)

If the question is "what does major/minor/patch mean exactly in Foreman/Katello/Satellite", then I have no good answer, as we do not follow SemVer, so really every version (3.11, 4.14, 6.16) is major, and these get patches (3.11.4, 4.14.3, 6.16.5), and there is not really a "minor" thing.

Reply:
Good point that we don't really follow semver